recipes/fs/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "fs" %}
+{% set version = "2.0.9" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "8570c6349db3b0bf68fe615fe2683ca04796082aceae0ae1d232f71a6f6e47ef" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - appdirs >=1.4.3,<2
+    - pytz
+    - six >=1.10.0,<2
+    - enum34 >=1.1.6,<2  # [py2k]
+    - scandir >=1.5,<2  # [py2k or py34]
+    
+  run:
+    - python
+    - appdirs >=1.4.3,<2
+    - pytz
+    - six >=1.10.0,<2
+    - enum34 >=1.1.6,<2  # [py2k]
+    - scandir >=1.5,<2  # [py2k or py34]
+
+test:
```

Comment:
So this section is tests? We run our tests with `tox`.

Reply:
it's a section of quick tests that an install succeeded.  It doesn't usually run a package's full test suite, because that could time out the continuous integration builds.  But if the test suite is quick, can add commands to quickly test that an install succeeded, beyond just that imports work.

Reply:
@willmcgugan is there a command that runs quickly (in a few minutes) and checks that an installed fs works correctly?

Reply:
None that I can think of. There is not a great deal to go wrong, tbh. If the import works, its probably safe to assume its installed.